kmod/src/forest.c

```diff
@@ -782,7 +791,7 @@ int scoutfs_forest_oino_alloc(struct super_block *sb, u64 trans_seq)
 	bb->hdr.magic = cpu_to_le32(SCOUTFS_BLOCK_MAGIC_BLOOM);
 	bb->hdr.fsid = SCOUTFS_SB(sb)->super.hdr.fsid;
 	bb->hdr.blkno = cpu_to_le64(blkno);
-	bb->hdr.seq = cpu_to_le64(trans_seq);
+	bb->hdr.seq = cpu_to_le64(0);
```

Comment:
We're setting hdr.seq to last_seq down in oino_update(), before put()ing the block. Since this value is overwritten before persistence maybe we shouldn't even be setting it here? if I'm understanding how this all works?

Reply:
ok so bloom.hdr.seq gets trans_seq and our new bloom.last_seq field gets last_seq.